tutorials/scripting/overridable_functions.rst

```diff
@@ -76,7 +76,7 @@ information, read the dedicated documentation:
 .. tabs::
  .. code-tab:: gdscript GDScript
 
-    # Called every frame, as often as possible.
```

Comment:
I think juxtaposing `as often as possible` here is confusing. It could be understood as `called on most frames, but not all`.